acorn/src/regexp.js

```diff
@@ -920,6 +981,249 @@ pp.regexp_eatClassEscape = function(state) {
   )
 }
 
+// https://tc39.es/ecma262/#prod-ClassSetExpression
+// https://tc39.es/ecma262/#prod-ClassUnion
+// https://tc39.es/ecma262/#prod-ClassIntersection
+// https://tc39.es/ecma262/#prod-ClassSubtraction
+pp.regexp_classSetExpression = function(state) {
+  let nextMayContainStrings = false
+  if (this.regexp_eatClassSetRange(state))
+    ;
```

Comment:
Please use brackets, not a semicolon, for empty branches.

Reply:
I fixed that.
https://github.com/acornjs/acorn/pull/1219/commits/8dd203ee7d96d24a6970e4f092d08a5f3f0249c5